src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx

```diff
@@ -82,11 +82,19 @@ const EnrolmentTable: React.FC<Props> = ({
 
   const approvedCount = enrolments
     .filter((e) => e.status === EnrolmentStatus.Approved)
-    .reduce((acc, cur) => acc + cur.studyGroup.groupSize, 0);
+    .reduce(
+      (acc, cur) =>
+        acc + cur.studyGroup.groupSize + cur.studyGroup.amountOfAdult,
+      0
+    );
 
   const pendingCount = enrolments
     .filter((e) => e.status === EnrolmentStatus.Pending)
-    .reduce((acc, cur) => acc + cur.studyGroup.groupSize, 0);
+    .reduce(
+      (acc, cur) =>
+        acc + cur.studyGroup.groupSize + cur.studyGroup.amountOfAdult,
+      0
+    );
```

Comment:
Could these be combined into a helper function that receives enrolments and EnrolmenStatus as params and returns the count?

Reply:
Amended! Moved filter-reduce to utils.